website/docs/r/rds_instance_v3.html.markdown

```diff
@@ -0,0 +1,259 @@
+---
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_rds_instance_v3"
+sidebar_current: "docs-opentelekomcloud-resource-rds-instance-v3"
+description: |-
+  instance management
+---
+
+# opentelekomcloud\_rds\_instance\_v3
+
+instance management
+
+## Example Usage
+
+### create a single db instance
+
+```hcl
+resource "opentelekomcloud_networking_secgroup_v2" "secgroup" {
+  name = "terraform_test_security_group"
+  description = "terraform security group acceptance test"
+}
+
+resource "opentelekomcloud_rds_instance_v3" "instance" {
+  availability_zone = "{{ availability_zone }}"
+  db = {
+    password = "Huangwei!120521"
+    type = "PostgreSQL"
+    version = "9.5"
+    port = "8635"
+  }
+  name = "terraform_test_rds_instance"
+  security_group_id = "${opentelekomcloud_networking_secgroup_v2.secgroup.id}"
+  network_id = "{{ network_id }}"
+  vpc_id = "{{ vpc_id }}"
+  volume {
+    type = "COMMON"
+    size = 100
+  }
+  flavor = "rds.pg.c2.medium"
+  backup_strategy = {
+    start_time = "08:00-09:00"
+    keep_days = 1
+  }
+}
+```
+
+### create a primary/standby db instance
+
+```hcl
+resource "opentelekomcloud_networking_secgroup_v2" "secgroup" {
+  name = "terraform_test_security_group"
+  description = "terraform security group acceptance test"
+}
+
+resource "opentelekomcloud_rds_instance_v3" "instance" {
+  availability_zone = "{{ availability_zone_1 }},{{ availability_zone_2 }}"
+  db = {
+    password = "Huangwei!120521"
+    type = "PostgreSQL"
+    version = "9.5"
+    port = "8635"
+  }
+  name = "terraform_test_rds_instance"
+  security_group_id = "${opentelekomcloud_networking_secgroup_v2.secgroup.id}"
+  network_id = "{{ network_id }}"
+  vpc_id = "{{ vpc_id }}" 
+  volume {
+    type = "COMMON"
+    size = 100
+  }
+  flavor = "rds.pg.s1.medium.ha"
+  ha_replication_mode = "async"
+  backup_strategy = {
+    start_time = "08:00-09:00"
+    keep_days = 1
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `availability_zone` -
+  (Required)
+  Specifies the AZ name. Changing this parameter will create a new resource.
+
+* `db` -
+  (Required)
+  Specifies the database information. Structure is documented below. Changing this parameter will create a new resource.
+
+* `flavor` -
+  (Required)
+  Specifies the specification code. Changing this parameter will create a new resource.
+
+* `name` -
+  (Required)
+  Specifies the DB instance name. The DB instance name of the same type
+  must be unique for the same tenant. The value must be 4 to 64
+  characters in length and start with a letter. It is case-sensitive
+  and can contain only letters, digits, hyphens (-), and underscores
+  (_).  Changing this parameter will create a new resource.
+
+* `network_id` -
```

Comment:
It would be good to mention here, that `network_id` is maped to the `subnet_id` parameter in OTC.

Reply:
I will rename 'network_id' to 'subnent_id'.

Reply:
Perfect and thank you for adding the section in the documentation!